.eslintrc.js

```diff
@@ -0,0 +1,40 @@
+var path = require('path');
+
+module.exports = {
+  env: {
+    es6: true,
+    browser: true,
+    node: true,
+  },
+  parserOptions: {
+    sourceType: 'module',
```

Comment:
the bytesForHumans module does use the ES2016 exponentiation operator, which is why ecmaVersion was set to 7 in the old config.

Reply:
The latest version of Buble does support this, but we should make sure buble is updated, and also polyfill Array.prototype.includes which is the other thing that made it into the ES2016 spec.